tests/x11/yast2_lan_restart.pm

```diff
@@ -66,8 +67,12 @@ sub change_hw_device_name {
 
     send_key 'alt-i';        # Edit NIC
     assert_screen 'yast2_lan_network_card_setup';
-    send_key 'alt-w';        # Hardware tab
-    assert_screen 'yast2_lan_hardware_tab';
+    if (is_sle('15-SP2+')) {
+        send_key 'alt-g';    # Starting with SLE15 SP2, "Device name" field is shown in General tab
```

Comment:
I'd would suggest adding after this line: `assert_screen 'yast2_lan_general_tab'` or we would end up with a transition `alt-g + alt-e` sporadically failing. WDYT?

Reply:
I did not add this assert_screen intentionally as I don't like the perspective to change the needle each time UI changes a bit. And also, if you'll look at ''yast2_lan_hardware_tab" needle that is [matching currently in openQA](https://openqa.suse.de/tests/3365437#step/yast2_lan_restart/168), it has lost its initial value as it simply matches the window header instead of the selected "Hardware" tab. So now it is almost useless after someone changed it that way (to, probably, make a quick fix).

The second thing is that even if after pressing 'alt-g', the tab will not be selected it anyways will fail on finding "Device name" field, as it is now:)

And, for example we are still sending the following keys without screen checks and it works fine:
``` 
send_key $cmd{ok};
    send_key $cmd{next};
```

Reply:
I don't like neither the approach, it is very painful maintain the needles. The problem is that I remember to fix this kind of transitions many times. Your verification is just isolating the test module, but in prod the VM is running many test module and sometimes is slower that we expect. But for simple change like this it does not worth to run statistics, let's try anyway to merge.